objects.js

```diff
@@ -0,0 +1,56 @@
+const createPerson = (name, age) => {
+  return {
+    name,
+    age,
+  };
+};
+
+const getName = (object) => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  return object.hasOwnProperty(property);
+};
+
+const isOver65 = (person) => {
+  if (person.age > 65) {
+    return true;
+  };
+};
+
+const getAges = (people) => {
+  // your code here
```

Comment:
For this one you can `map()` over the `people` array to produce an new array.

eg:
```
return people.map(person => {
  return person.age;
})
```